velox/dwio/parquet/tests/ParquetTableScanTest.cpp

```diff
@@ -86,3 +91,25 @@ TEST_F(ParquetTableScanTest, basic) {
 
   assertQuery(plan, {split}, "SELECT min(a), max(b) FROM tmp");
 }
+
+TEST_F(ParquetTableScanTest, emptyProjection) {
+  auto data = makeRowVector(
+      {"a", "b"},
+      {
+          makeFlatVector<int64_t>(20, [](auto row) { return row + 1; }),
+          makeFlatVector<double>(20, [](auto row) { return row + 1; }),
+      });
+  createDuckDbTable({data});
+
+  auto filePath = getExampleFilePath("sample.parquet");
+  auto split = makeSplit(filePath);
+
+  // Projection does not have any columns
+  auto rowType = ROW({}, {});
+  auto plan = PlanBuilder()
+                  .tableScan(rowType)
+                  .singleAggregation({}, {"count(0)"})
+                  .planNode();
+
+  assertQuery(plan, {split}, "SELECT count(*) FROM tmp");
```

Comment:
nit: This test doesn't really need to create a table in DuckDB as it can simply assertQuery(plan, {split}, "SELECT 20")